source/mainnet/net/voting/guardians.rst

```diff
@@ -0,0 +1,193 @@
+
+    .. image:: ../images/voting/guardian-generate-secret-key-share.png
+        :alt: generate secret key share dialog
+        :width: 50%
+
+#. Click **Register validation OK**.
+
+Once you have generated your secret key share you must wait until the other guardians have done the same.
+
+.. image:: ../images/voting/guardian-await-key-share.png
+    :alt: dialog shown when awaiting other guardians
+    :width: 50%
+
+After all guardians have complete the process, a screen shows that election setup is complete and a countdown to the election start.
+
+.. image:: ../images/voting/guardian-election-setup-complete.png
+    :alt: dialog shown when election setup is completed by all guardians
+    :width: 50%
+
+If corruption is detected the guardian should register a complaint in the contract.
+
+After the election
+==================
+
+The app retrieves the encrypted tally from the contract automatically.
+
+Generate your decrypted share
+-----------------------------
+
+#. Open the Guardian app.
+
+#. Click **Generate decryption share**.
+
+    .. image:: ../images/voting/guardian-generate-decryption-share.png
+        :alt: generate decryption share dialog
+        :width: 50%
+
+#. Click **Send share registration**.
+
+    .. image:: ../images/voting/guardian-register-decryption-share.png
+        :alt: register decryption share dialog
+        :width: 50%
+
+Once you have registered your decryption share you must wait until the other guardians have done the same.
+
+.. image:: ../images/voting/guardian-await-decryption.png
+    :alt: dialog shown when awaiting other guardians
+    :width: 50%
+
+Register the decrypted share in the contract
+--------------------------------------------
+
+The final step creates a proof that can be checked by others to determine that the election has been fair.
+
+#. Click **Generate decryption proof**.
+
+    .. image:: ../images/voting/guardian-generate-decryption-proof.png
+        :alt: generate decryption proof dialog
+        :width: 50%
+
+#. Click **Send proof registration**.
+
+    .. image:: ../images/voting/guardian-register-decryption-proof.png
+        :alt: generate decryption proof dialog
+        :width: 50%
+
+Once you have registered your decryption proof you must wait until the other guardians have done the same.
+
+.. image:: ../images/voting/guardian-await-decryption-proof.png
+    :alt: dialog shown when awaiting other guardians
+    :width: 50%
+
+After all guardians have registered decryption proofs, the decryption is shown as complete.
+
+.. image:: ../images/voting/guardian-decryption-complete.png
+    :alt: dialog shown when awaiting other guardians
+    :width: 50%
+
+Uninstall the app
+=================
+
+Once the election is final, guardians should uninstall the app. The instructions below describe how to uninstall the app for each platform.
+
+Windows
+-------
+
+Uninstall the app as you uninstall all apps on Microsoft.
+
+Delete the file XXXX to remove all guardian keys.
```

Comment:
```suggestion
Delete the folder ``%appdata%\com.concordium.guardian.<network>.<contract-index>.<contract-subindex>`` to remove all guardian keys.
```

@mh-concordium can you verify that this is correct?

An alternative path description could be ``C:\Users\<user>\AppData\com.concordium.guardian.<network>.<contract-index>.<contract-subindex>``